src/morphosource/config.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+API_URL = os.environ.get("MORPHOSOURCE_API_URL", "https://www.morphosource.org/api")
```

Comment:
Just a curiosity, why's this set to check for an environment variable first?

Reply:
In case there is ever a dev instance of MorphoSource that we need to test against.